src/sites/dMCDA.jl

```diff
@@ -37,6 +37,65 @@ struct DMCDA_vars  # {V, I, F, M} where V <: Vector
     wtzonesshade # ::F
 end
 
+"""
+    DMCDA_vars(domain::Domain, criteria::NamedVector, site_ids::AbstractArray, sumcover::AbstractArray, area_to_seed::Float64)::DMCDA_vars
+ 
+Constuctor for DMCDA variable type, where criteria are defined as a NamedVector.
+"""
+function DMCDA_vars(domain::Domain, criteria::NamedVector, site_ids::AbstractArray, sumcover::AbstractArray, area_to_seed::Float64)::DMCDA_vars
+    # Site Data
+    site_d = domain.site_data
+    nsites = size(site_d, 1)
```

Comment:
No action needed, but this makes me think we should also support `n_sites(domain)`...

Reply:
Yeah this is probably a good idea